app/views/handlers/list_action.py

```diff
@@ -15,9 +15,12 @@ def parent_location(self):
             block_id=self.rendered_block["parent_id"],
         )
 
+    def _get_routing_path(self):
+        return self.router.routing_path(section_id=self.parent_location.section_id)
+
     def is_location_valid(self):
         can_access_parent_location = self.router.can_access_location(
-            self.parent_location, self._routing_path
+            self.parent_location, self._get_routing_path()
```

Comment:
Calling `self._get_routing_path` will regenerate the routing path again, did this not work just calling `self._routing_path`?

This applies to all places this occurs.